R/commonMachineLearningClassification.R

```diff
@@ -15,254 +15,236 @@
-    classificationTable$addColumnInfo(name = 'distance', title = gettext('Distance'),          type = 'string')
-
-  } else if (type =="lda") {
-
-    classificationTable$addColumnInfo(name = 'lda',    title = gettext('Linear Discriminants'), type = 'integer')
-    classificationTable$addColumnInfo(name = 'method', title = gettext('Method'),               type = 'string')
-
+    table$addColumnInfo(name = "nn", title = gettext("Nearest neighbors"), type = "integer")
+    table$addColumnInfo(name = "weights", title = gettext("Weights"), type = "string")
+    table$addColumnInfo(name = "distance", title = gettext("Distance"), type = "string")
+  } else if (type == "lda") {
+    table$addColumnInfo(name = "lda", title = gettext("Linear Discriminants"), type = "integer")
+    table$addColumnInfo(name = "method", title = gettext("Method"), type = "string")
   } else if (type == "randomForest") {
-
-    classificationTable$addColumnInfo(name = 'trees', title = gettext('Trees'),                type = 'integer')
-    classificationTable$addColumnInfo(name = 'preds', title = gettext('Predictors per split'), type = 'integer')
-
+    table$addColumnInfo(name = "trees", title = gettext("Trees"), type = "integer")
+    table$addColumnInfo(name = "preds", title = gettext("Predictors per split"), type = "integer")
   } else if (type == "boosting") {
-
-    classificationTable$addColumnInfo(name = 'trees',     title = gettext('Trees'),     type = 'integer')
-    classificationTable$addColumnInfo(name = 'shrinkage', title = gettext('Shrinkage'), type = 'number')
-
+    table$addColumnInfo(name = "trees", title = gettext("Trees"), type = "integer")
+    table$addColumnInfo(name = "shrinkage", title = gettext("Shrinkage"), type = "number")
   } else if (type == "neuralnet") {
-
-    classificationTable$addColumnInfo(name = 'layers',     title = gettext('Hidden Layers'),     type = 'integer')
-    classificationTable$addColumnInfo(name = 'nodes',     title = gettext('Nodes'),     type = 'integer')
-
+    table$addColumnInfo(name = "layers", title = gettext("Hidden Layers"), type = "integer")
+    table$addColumnInfo(name = "nodes", title = gettext("Nodes"), type = "integer")
   }
-
   # Add common columns
-  classificationTable$addColumnInfo(name = 'ntrain', title = gettext('n(Train)'), type = 'integer')
-  if(options[["modelOpt"]] != "optimizationManual")
-    classificationTable$addColumnInfo(name = 'nvalid', title = gettext('n(Validation)'), type = 'integer')
-  classificationTable$addColumnInfo(name = 'ntest', title = gettext('n(Test)'), type = 'integer')
-  if(options[["modelOpt"]] != "optimizationManual")
-    classificationTable$addColumnInfo(name = 'validAcc', title = gettext('Validation Accuracy'), type = 'number')
-  classificationTable$addColumnInfo(name = 'testAcc', title = gettext('Test Accuracy'), type = 'number')
-
+  table$addColumnInfo(name = "ntrain", title = gettext("n(Train)"), type = "integer")
```

Comment:
wouldn't nTrain be more consistent with the other camelCased names such as testAcc, validAcc, optimizationManual.